modules/items/server.lua

```diff
@@ -37,6 +37,44 @@ local trash = {
 	{description = 'An empty chips bag.', weight = 5, image = 'trash_chips'},
 }
 
+local imageWhitelist = {
+	hosts = {"imgur.com"},
+	fileTypes = {".png", ".apng"}
```

Comment:
Probably better to use a set here, rather than array. Also need to use `'` for strings.